client/src/components/ActiveChat/Messages.js

```diff
@@ -5,10 +5,11 @@ import moment from "moment";
 
 const Messages = (props) => {
   const { messages, otherUser, userId } = props;
+  let tempMessages = messages.slice().reverse()
```

Comment:
This is inefficient because it will happen every time the component re-renders. It would be better to sort the messages just once.